template/{{.input_root_dir}}/.azure/devops-pipelines/{{.input_project_name}}-bundle-cicd.yml.tmpl

```diff
@@ -10,8 +10,12 @@ trigger:
     include:
       - {{template `default_branch` .}}
       - {{template `release_branch` .}}
+  paths:
+    include:
+      - {{template `project_name_alphanumeric_underscore` .}}/*
 
 variables:
+  - workingDirectory: {{template `project_name_alphanumeric_underscore` .}}
```

Comment:
@niall-turbitt to test ADO changes. Would you mind also generating and testing a monorepo setup with ADO to confirm it works/triggers as expected?

Reply:
if you're adding workingDirectory as another variable, the syntax needs to be changed here. It should be:

```
variables:
  - name: workingDirectory
     value: {{template `project_name_alphanumeric_underscore` .}}
  - group: {{template `project_name` .}} variable group 
```

Otherwise it results in the following error ![Screenshot 2024-01-11 at 11 09 03](https://github.com/databricks/mlops-stacks/assets/52167569/d93823c7-706e-432d-870c-26fcd196334c)

Reply:
Thanks for calling out, fixed! Anything else that needs to be changed for ADO?

Reply:
everything else looks good! Tested out on [this repo](https://dev.azure.com/niallturbitt/_git/mono-mlops-stacks)